pybossa/cache/__init__.py

```diff
@@ -123,6 +125,9 @@ def cache(key_prefix, timeout=300, cache_group_keys=None):
         timeout = DEFAULT_TIMEOUT
     elif timeout < MIN_TIMEOUT:
         timeout = MIN_TIMEOUT
+
+    timeout += randrange(30)  # add a random jitter to reduce DB load
```

Comment:
We have scheduled jobs to "refresh" caches. They are executed almost at the same time with same TTL. Once it expires, it will expires at almost the same time. And the DB load might be burst at this time. By adding a random jitter, it reduces the possibility the cache to be expired at the same time and balanced the DB load to avoid many requests hitting the DB.

Reply:
Thanks for clarification. I recommend adding your comments to the code as well.

Reply:
Will do.